stdlib/float.mli

```diff
@@ -433,32 +440,33 @@ module Array : sig
   (** Same as {!append}, but concatenates a list of floatarrays. *)
 
   val sub : t -> int -> int -> t
-  (** [sub a start len] returns a fresh floatarray of length [len],
-      containing the elements number [start] to [start + len - 1]
+  (** [sub a pos ~len] returns a fresh floatarray of length [len],
```

Comment:
The alpha-renaming is fine, but `~` marks have appeared throughout the unlabeled version

Reply:
Fixed.